shared/django_apps/dummy_settings.py

```diff
@@ -8,18 +8,63 @@
 
 # Install apps so that you can make migrations for them
 INSTALLED_APPS = [
+    "shared.django_apps.legacy_migrations",
     "shared.django_apps.pg_telemetry",
     "shared.django_apps.ts_telemetry",
     "shared.django_apps.rollouts",
+    # API models
+    "django.contrib.admin",
+    "django.contrib.auth",
+    "django.contrib.contenttypes",
+    "django.contrib.messages",
+    "django.contrib.postgres",
+    "shared.django_apps.codecov_auth",
+    "shared.django_apps.core",
+    "shared.django_apps.reports",
 ]
 
-MIDDLEWARE = []
+# Migrated from API to get Minio working
+MINIO_ACCESS_KEY = get_config("services", "minio", "access_key_id")
+MINIO_SECRET_KEY = get_config("services", "minio", "secret_access_key")
+MINIO_LOCATION = "codecov.s3.amazonaws.com"
+MINIO_HASH_KEY = get_config("services", "minio", "hash_key")
+ARCHIVE_BUCKET_NAME = "codecov"
+
+MIDDLEWARE = [
+    "django.contrib.auth.middleware.AuthenticationMiddleware",
+    "django.middleware.common.CommonMiddleware",
+    "django.middleware.csrf.CsrfViewMiddleware",
+    "django.contrib.messages.middleware.MessageMiddleware",
+    "django.contrib.sessions.middleware.SessionMiddleware",
+]
+
+# Migrated from API
+TEMPLATES = [
+    {
+        "BACKEND": "django.template.backends.django.DjangoTemplates",
+        "DIRS": [],
+        "APP_DIRS": True,
+        "OPTIONS": {
+            "context_processors": [
+                "django.template.context_processors.debug",
+                "django.template.context_processors.request",
+                "django.contrib.auth.context_processors.auth",
+                "django.contrib.messages.context_processors.messages",
+            ]
+        },
+    }
```

Comment:
is this still necessary?

Reply:
Deleted and ran test, no errors. I'll rebuild API with this image tomorrow and test